.travis.yml

```diff
@@ -12,6 +12,7 @@ jdk:
 script:
   # tests are not done with mvn test - dont know why
   - ./scripts/build_mvn.sh
+  - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make fintech-ui-test
```

Comment:
looks like travis is using other angular version than expected. Build fails due to this:

Makefile:76: recipe for target 'fintech-ui-test' failed
make: *** [fintech-ui-test] Error 1
travis_time:end:1cf331f5:start=1582282152590755213,finish=1582282212945483794,duration=60354728581,event=script
[0K[31;1mThe command "docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make fintech-ui-test" exited with 2.[0m

Reply:
fix the failure